src/app/client/src/app/modules/public/module/explore/components/explore/explore.component.ts

```diff
@@ -92,7 +92,7 @@ export class ExploreComponent implements OnInit, OnDestroy, AfterViewInit {
       takeUntil(this.unsubscribe$))
       .subscribe((result) => {
         this.showLoader = true;
-        this.queryParams = { ...result[0], ...result[1] };
+        this.queryParams = { ...result[1] };
```

Comment:
If "slug" param is causing an issue, check for slug object to omit. Present code is blindly relaying on index, later if order changed then also it will fail.